src/main/scala/org/phenoscape/kb/KB.scala

```diff
@@ -189,8 +190,44 @@ OPTIONAL {
     App.executeSPARQLQueryString(query.text, res => Instant.parse(res.getLiteral("date").getLexicalForm)).map(_.head)
   }
 
+  def getKBMetadata = {
+    val builtFut = buildDate
+    val ontologiesFut = kbOntologies
+    for {
+      built <- builtFut
+      ontologies <- ontologiesFut
+    } yield KBMetadata(built, ontologies)
+  }
+
+  def kbOntologies = {
+    val query = sparql"""
+    PREFIX owl: <http://www.w3.org/2002/07/owl#>
+    SELECT ?ont ?version
+    FROM $KBMainGraph
+    WHERE {
+      ?ont $rdfType owl:Ontology .
+  	  ?ont owl:versionIRI ?version
+    }
+  """
+
+    App.executeSPARQLQueryString(query.text, qs => (IRI.create(qs.getResource("ont").getURI), IRI.create(qs.getResource("version").getURI)))
+  }
 }
 
+case class KBMetadata(built: Instant, ontologies: Seq[(IRI, IRI)]) extends JSONResultItem {
+
+  def toJSON: JsObject = Map(
+    "build_time" -> built.toString.toJson,
+    "ontologies" -> ontologies.map{
+      case(ont, version) => JsObject(
+        "ontololgy" -> ont.toString.toJson,
```

Comment:
Typo

Reply:
To go along with our JSON-LD-ish convention we use elsewhere, use key `@id` instead of `ontology`.

Reply:
> Typo

Where?

Reply:
"ontololgy". But as I said I prefer not to have this key anyway.